### DISCUSSION TITLE
Does `take_over_my_home` work in other vaults?

### DISCUSSION THREAD

**Discussion by jrmallorca** [nvim-telekasten/telekasten.nvim#209] [Q&A]
I have put my telekasten mappings into **ftplugin/telekasten.lua** and have set `take_over_my_home` and `auto_set_filetype` to `true` in my other vault configuration.

When I open any of my other vault's notes NOT through the `switch_vault` functionality, then the command `:set filetype?` outputs `markdown` and vice versa.

This means that I have to keep opening notes from my other vault through the `switch_vault` functionality to make the filetypes `telekasten`. I was wondering if I could avoid this.

**Comment by lambtho12** [comment]
Is it something you noticed a few days ago or is it a new behavior?

See this here https://github.com/renerocksai/telekasten.nvim/issues/208#issuecomment-1404899502, I also have issues with the filetype for some notes but not other.

**Reply by jrmallorca** [reply]
Unfortunately, I've only started using this plugin recently but this behaviour has probably been present for me since 4 days ago (when I first started using it).

**Reply by lambtho12** [reply]
I am fairly busy at the moment, but this is definitely something that should be sorted out.
The whole filetype/syntax stuff can probably be improved quite a lot to be honest...
I will try to have a look next week.

**Reply by jrmallorca** [reply]
Thank you! This plugin is a great replacement and allows me to create better notes through neovim so I appreciate its creation and maintenance by you guys.
